src/app/material.module.ts

```diff
@@ -0,0 +1,42 @@
+import {NgModule} from '@angular/core';
+import {
+  MdButtonModule,
+  MdCardModule,
+  MdDialogModule,
+  MdIconModule,
+  MdInputModule,
+  MdMenuModule,
+  MdOptionModule,
+  MdSidenavModule,
+  MdSlideToggleModule,
+  MdToolbarModule
+} from '@angular/material';
+
+@NgModule({
+  imports: [
+    MdButtonModule,
+    MdCardModule,
+    MdDialogModule,
+    MdIconModule,
+    MdInputModule,
+    MdMenuModule,
+    MdOptionModule,
+    MdSidenavModule,
+    MdSlideToggleModule,
+    MdToolbarModule
+  ],
+  exports: [
+    MdButtonModule,
+    MdCardModule,
+    MdDialogModule,
+    MdIconModule,
+    MdInputModule,
+    MdMenuModule,
+    MdOptionModule,
```

Comment:
Can you remove the whole `imports` section (we only need to export here) and also remove the `MdOptionModule` here. The `MdOptionModule` should be included by the `MdMenuModule` and `MdSelectModule` automatically.